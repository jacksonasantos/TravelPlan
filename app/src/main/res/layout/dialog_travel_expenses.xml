<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="fill_parent"
    android:layout_height="fill_parent"
    android:orientation="vertical"
    android:background="@color/colorBackgroundDialog"
    android:padding="10dp">
    <androidx.cardview.widget.CardView
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:cardBackgroundColor="@color/colorBackgroundCard"
        app:cardCornerRadius="@dimen/cardCornerRadius"
        app:cardElevation="@dimen/cardElevation">
        <LinearLayout
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:orientation="vertical"
            android:padding="@dimen/labels_margin">

            <TextView
                android:id="@+id/txtExpenseType"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:text="@string/TravelExpenses_ExpenseType"
                android:textAppearance="?android:attr/textAppearanceMedium"
                android:textSize="16sp" />

            <Spinner
                android:id="@+id/spinExpenseType"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:entries="@array/expenses_type_array"
                android:textSize="16sp"
                android:text="@string/TravelExpenses_ExpenseType">
            </Spinner>

            <TextView
                android:id="@+id/txtExpectedValue"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:text="@string/TravelExpenses_ExpectedValue"
                android:textAppearance="?android:attr/textAppearanceMedium"
                android:textSize="16sp" />

            <EditText
                android:id="@+id/etExpectedValue"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:autofillHints="no"
                android:inputType="numberDecimal"
                android:textSize="16sp"
                tools:ignore="LabelFor" />

            <TextView
                android:id="@+id/txtNote"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:text="@string/TravelExpenses_Note"
                android:textAppearance="?android:attr/textAppearanceMedium"
                android:textSize="16sp" />
            <EditText
                android:id="@+id/etNote"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:autofillHints="no"
                android:inputType="textCapSentences"
                android:textSize="16sp"
                tools:ignore="LabelFor" />
        </LinearLayout>
    </androidx.cardview.widget.CardView>
</LinearLayout>