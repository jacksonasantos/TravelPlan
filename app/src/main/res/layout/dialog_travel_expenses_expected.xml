<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="fill_parent"
    android:layout_height="fill_parent"
    android:orientation="vertical"
    android:background="@color/colorBackgroundDialog"
    android:padding="10dp">
    <androidx.cardview.widget.CardView
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_marginTop="@dimen/labels_space_top"
        android:layout_marginBottom="@dimen/labels_space_top"
        app:cardBackgroundColor="@color/colorBackgroundCard"
        app:cardCornerRadius="@dimen/cardCornerRadius"
        app:cardElevation="@dimen/cardElevation">
        <LinearLayout
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:orientation="vertical"
            android:padding="@dimen/labels_margin">
            <TextView
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:gravity="center"
                android:text="@string/TravelExpenseExpected"
                android:textColor="@color/design_default_color_secondary_variant"
                android:textSize="14sp"/>
            <LinearLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:orientation="horizontal">
                <TextView
                    android:id="@+id/txtExpenseType"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_weight="0.30"
                    android:text="@string/TravelExpenses_ExpenseType"
                    android:textAppearance="?android:attr/textAppearanceMedium"
                    android:textSize="14sp"  />
                <Spinner
                    android:id="@+id/spinExpenseType"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_weight="0.70"
                    android:entries="@array/expenses_type_array"
                    android:textSize="14sp"
                    android:text="@string/TravelExpenses_ExpenseType">
                </Spinner>
            </LinearLayout>
            <LinearLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:orientation="horizontal">
                <TextView
                    android:id="@+id/txtExpectedValue"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_weight="0.30"
                    android:text="@string/TravelExpenses_ExpectedValue"
                    android:textAppearance="?android:attr/textAppearanceMedium"
                    android:textSize="14sp" />
                <EditText
                    android:id="@+id/etExpectedValue"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_weight="0.70"
                    android:gravity="end"
                    android:autofillHints="no"
                    android:inputType="numberDecimal"
                    android:textSize="14sp"
                    tools:ignore="LabelFor" />
            </LinearLayout>
            <LinearLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:orientation="horizontal">
                <TextView
                    android:id="@+id/txtNote"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_weight="0.30"
                    android:text="@string/TravelExpenses_Note"
                    android:textAppearance="?android:attr/textAppearanceMedium"
                    android:textSize="14sp"  />
                <EditText
                    android:id="@+id/etNote"
                    android:layout_width="0dp"
                    android:layout_height="wrap_content"
                    android:layout_weight="0.70"
                    android:autofillHints="no"
                    android:inputType="textCapSentences"
                    android:textSize="14sp"
                    tools:ignore="LabelFor" />
            </LinearLayout>
        </LinearLayout>
    </androidx.cardview.widget.CardView>
</LinearLayout>